### PR TITLE
project declaration

### PR DESCRIPTION
**Pull request by Guis99**
**Name**: Brian Hoh

**CV Link**: https://drive.google.com/file/d/1G2_3NeoKxAKfBFPg6U2hKE0Nc-C21GRn/view

**About me**: Hello! I graduated MIT in 2022 with a degree in aerospace engineering, and I currently work full-time at MathWorks as a software engineer, where I write mostly MATLAB, as well as some JavaScript and C++. I am interested in contributing to open source software in the space of scientific computing and numerical analysis and have explored these fields through a number of personal projects in my spare time.

My recent projects can be found on my Github: https://github.com/Guis99

Specifically, I would like to draw attention to three repositories:

https://github.com/Guis99/learnJulia - As the title suggests, I've been teaching myself Julia through the use of this repo, where I implemented a basic Finite Element program based on one of my older projects.

https://github.com/Guis99/DisGal - My implementation of several partial differential equation solvers using the discontinuous galerkin method (most up-to-date branch is 'toBeRefactored')

https://github.com/Guis99/MathParser - A tool written in C++ for parsing and evaluating strings as mathematical expressions. 

My background is mostly in numerical methods, but am happy to do some reading on symbolic computing and PINNs, as well as review the codebase in question. This community is at a unique intersection of my interests, and I intend to learn the background material required to successfully contribute to it. 

**Project description**: NeuralPDE is a package for training Physics Informed Neural Networks (PINNs) uses a symbolic representation for problems which are then lowered into generated code for the user, fully automating the PINN experience. However, the previous formulation of the codegen process used pre-built kernels that can be difficult to debug. The goal of this project is to rewrite the loss function generator to instead generate a Symbolics.jl expression, which can be analyzed a generate code through build_function.

### CODE REVIEW

**Review by ChrisRackauckas — APPROVED** [review]
Needs 2 more approvals

**Review by isaacsas — APPROVED** [review]
